src/component/BreadCrumb.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {Breadcrumbs, Typography} from "@mui/material";
+
+function BreadCrumb({data}) {
```

Comment:
we don't need data prop. We can directly have values array as prop. 

so here it will become: `function BreadCrumb({values}) {`

Reply:
Updated Breadcrumb with values as prop